cmd/collector/app/handler/zipkin_receiver_tls_test.go

```diff
@@ -12,59 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package server
+package handler
```

Comment:
This was testing exhaustively TLS settings on Zipkin endpoint, so I decided to keep it, just move from a different location. It works against the Zipkin receiver now.